src/types/codes/http_response_codes.ts

```diff
@@ -13,5 +13,4 @@ export enum DiscordHTTPResponseCodes {
   GatewayUnavailable = 502,
 }
 
-export type HTTPPResponseCodes = DiscordHTTPResponseCodes;
-export const HTTPPResponseCodes = DiscordHTTPResponseCodes;
+export { DiscordHTTPResponseCodes as HTTPResponseCodes };
```

Comment:
```suggestion
export type HTTPResponseCodes = DiscordHTTPResponseCodes;
export const HTTPResponseCodes = DiscordHTTPResponseCodes;
```

Reply:
Any reason as to why this is necessary?

Reply:
Skillz decided to use this to export enum aliases